python/ray/autoscaler/updater.py

```diff
@@ -324,7 +342,7 @@ def check_container_status(self):
             cmd, with_output=True).decode("utf-8").strip()
         if no_exist in output:
             return False
-        return output
+        return "true" in output.lower()
```

Comment:
was this an existing bug? should be covered by the `no_exist in output` check right?

Reply:
Yeah, in the case that the container exists but is not running, `check_container_status` returns `False`, not `"false"`. The latter is a `True` value and would break checks like:
```
if self.check_container_status()
```